src/api/environment.cc

```diff
@@ -436,6 +436,14 @@ Environment* GetCurrentEnvironment(Local<Context> context) {
   return Environment::GetCurrent(context);
 }
 
+IsolateData* GetEnvironmentIsolateData(Environment* env) {
+  return env->isolate_data();
+}
+
```

Comment:
for my understanding, what would be a use case for embedders with`IsolateData` instance?

Reply:
Using the same `IsolateData` instance for two different `Environment`s, or checking whether we’re using the Node.js `ArrayBuffer::Allocator`.